CHANGELOG.md

```diff
@@ -2,7 +2,17 @@
 
 BUG FIXES:
 
+  * connection/ssh: fix issue on machines with an SSH Agent available
+    preventing `key_file` from being read without explicitly
+    setting `agent = false` [GH-2615]
+  * core: don't prompt for variables with defaults [GH-2613]
+  * core: Return correct number of planned updates [GH-2620]
   * providers/aws: Allow uppercase characters in `aws_elb.name` [GH-2580]
+  * providers/aws: Allow underscores in `aws_db_subnet_group.name` (undocumented by AWS) [GH-2604]
+
+IMPROVEMENTS:
+
+  * provider/aws: Create RDS databases from snapshots [GH-2062]
```

Comment:
We leave off CHANGELOG updates on PRs to avoid conflicts - the maintainers will update this post-merge.